.circleci/config.yml

```diff
@@ -33,40 +56,121 @@ jobs:
           name: nose2
           command: pipenv run nose2 -v
 
-  build-wheel:
+  build-api-doc:
     docker:
       - image: python:3-buster
     steps:
       - checkout
       - install-python-dev-tools
       - run:
-          name: Make wheel
+          name: sphinx-build
           command: |
-            pipenv run python setup.py bdist_wheel
+            pipenv run sphinx-build docs build/sphinx/html
+      - persist_to_workspace:
+          root: build/sphinx
+          paths:
+            - html
```

Comment:
plz keep api docs as artifacts by `store_artifacts`

Reply:
for what reason?
want to check docs out of artifacts before merging?

Reply:
To make docs reviewable for PR reviewers without rebuilding it on their own environment.

Reply:
got it.

Reply:
3a2b628